tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -253,8 +246,8 @@ public function testcreateAndGetHonorContact() {
     $this->assertEquals(array(
       $id => array(
         'honor_type' => 'In Honor of',
-        'honorId' => $id,
-        'display_name' => 'John Doe',
+        'honorId' => $contactId,
+        'display_name' => 'Mr. John Doe II',
```

Comment:
The display name differs a bit as a result of the different create method. I decided not to fight it